### DISCUSSION TITLE
Duplicate Alert Button

### DISCUSSION THREAD

**Discussion by atronia** [grafana/grafana#53406] [Alerting]
Like the ability to duplicate Dashboards in grafana, by saving has...
There should also be a mechanism to duplicate existing alerts, that target other querys but the mechanism of detection is the same.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
